GliaWidgets/Component/Button/Header/HeaderButtonStyle.swift

```diff
@@ -1,9 +1,18 @@
 import UIKit
 
+/// Style of a header button.
 public struct HeaderButtonStyle {
+    /// Image of the button.
```

Comment:
Just like with any `UIImage` in the Widgets, it can be whatever the Integrator wishes for - FullHD Twilight Sparkle from My Little Pony or a 2x2 black dot. If the image asset is marked with "Render as: Template image", it will be possible to change its color. All images should resize automatically, so the Integrator doesn't have to worry about the correct size or aspect ratio.

This is true for all images/icons. I think mentioning it everywhere is too much, to be fair.
Not sure if anything needs to be changed here. Feel free to disagree and propose your own ideas 😃

Reply:
I guess if we add to the general guidelines that the images need to be "Render as: Template image" and they will be resized automatically, this would be enough from technical side.
From the look and feel side, they would need to try it out and consult with their own designers anyway, probably.